cra-rxjs-styled-components/src/components/repo-issues/Issues/Issues.data.tsx

```diff
@@ -0,0 +1,18 @@
+import { useState } from 'react';
```

Comment:
Why are we calling this file `Issues.data.tsx`? I don't think we are doing anything with data here. 
We can simply call it `Issues.tsx` IMO